src/ontology/ecto.Makefile

```diff
@@ -153,4 +153,20 @@ tmp/mre_seed.txt:
 	
 
 mre: tmp/mre_seed.txt
-	$(ROBOT) filter -i $(SRC) -T tmp/mre_seed.txt -o $@
\ No newline at end of file
+	$(ROBOT) filter -i $(SRC) -T tmp/mre_seed.txt -o $@
+
+
+# ----------------------------------------
+# NANOBOT
+# ----------------------------------------
+
+_nanobot:
+	chmod +x $@
+
+NANOBOT := ./nanobot
+
+### Databases
+
+.PHONY: clean
+clean:
```

Comment:
`clean` already exists in the makefile, use this instead:

```
. PHONY: clean_nanobot

clean_nanobot:
	rm -rf .nanobot.db nanobot

clean: clean_nanobot
```

Also better not use `r` unless you mean to delete a directory.

Reply:
done